CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * [#2975](https://github.com/bbatsov/rubocop/issues/2975): Make comment indentation before `)` consistent with comment indentation before `}` or `]`. ([@jonas054][])
 * [#3010](https://github.com/bbatsov/rubocop/issues/3010): Fix double reporting/correction of spaces after ternary operator colons (now only reported by `Style/SpaceAroundOperators`, and not `Style/SpaceAfterColon` too). ([@owst][])
 * [#3006](https://github.com/bbatsov/rubocop/issues/3006): Handle non-local variables as receivers inside `each_with_object` in `Performance/RedundantMerge`. ([@lumeet][])
+* [#3006](https://github.com/bbatsov/rubocop/issues/3006): Register an offense for non-local variables as receivers inside `eache_with_object` in `Performance/RedundantMerge`. ([@rrosenblum][])
```

Comment:
`eache_with_object` :) I guess you could mention element references here instead of non-local variables.